SAPHana/man/SAPHanaSrMultiTarget.py.7

```diff
@@ -321,7 +321,7 @@ the sudo permission configuration
 the directory with HANA trace files
 .TP
 /usr/sap/$SID/HDB$nr/.crm_attribute.$SITE
-the internal cache for srHook status changes while Linux cluster is down, file is owned by ${SID}adm and must never be touched
+the internal cache for srHook status changes while Linux cluster is down, file is owned and r/w by ${sid}adm and must never be touched
```

Comment:
it would be more clear and consistent (see line 343 in this PR) to use 'read/write' instead of 'r/w'

Reply:
done